content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -18,12 +18,12 @@ document better and be able to customize based on your needs.
 These are common one time setups you need for all the instructions below:
 
 ### Download kubectl CLI tool
-You can get kubectl from [its official doc](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+Follow the [kubectl installation guide].(https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```

Comment:
The link doesn't render properly at the moment. See preview:
https://deploy-preview-1402--competent-brattain-de2d6d.netlify.com/docs/pipelines/standalone-deployment-gcp/

To fix the problem, remove the full stop after the second square bracket.